pages/git-protocol.md

```diff
@@ -21,7 +23,9 @@ A guide for programming within version control.
 * Perform work in a feature branch.
 * Rebase frequently to incorporate upstream changes.
 * Use a [pull request] for code reviews.
+* [Sign your commits with a GPG key](https://help.github.com/categories/gpg/).
 
+[2FA]: https://help.github.com/articles/about-two-factor-authentication/
```

Comment:
This initially threw me off because I thought you had a link to 2FA twice, and one wasn't showing up in the list when viewing the file.  I see it's a shortcut link now, but wanted to just call out the fact that at a glance it's easy to mistake it when looking at the markdown itself.